lib/core/utils/get-flattened-tree.js

```diff
@@ -119,59 +131,52 @@ function flattenTree(node, shadowId, parent) {
     retVal = createNode(node, parent, shadowId);
     shadowId = 'a' + Math.random().toString().substring(2);
     realArray = Array.from(node.shadowRoot.childNodes);
-    retVal.children = realArray.reduce((res, child) => {
-      return reduceShadowDOM(res, child, retVal);
-    }, []);
+    retVal.children = createChildren(realArray, retVal, shadowId);
 
     return [retVal];
-  } else {
-    if (
-      nodeName === 'content' &&
-      typeof node.getDistributedNodes === 'function'
-    ) {
-      realArray = Array.from(node.getDistributedNodes());
-      return realArray.reduce((res, child) => {
-        return reduceShadowDOM(res, child, parent);
-      }, []);
-    } else if (
-      nodeName === 'slot' &&
-      typeof node.assignedNodes === 'function'
-    ) {
-      realArray = Array.from(node.assignedNodes());
-      if (!realArray.length) {
-        // fallback content
-        realArray = getSlotChildren(node);
-      }
-      const styl = window.getComputedStyle(node);
-      // check the display property
-      if (false && styl.display !== 'contents') {
-        // intentionally commented out
-        // has a box
-        retVal = createNode(node, parent, shadowId);
-        retVal.children = realArray.reduce((res, child) => {
-          return reduceShadowDOM(res, child, retVal);
-        }, []);
-
-        return [retVal];
-      } else {
-        return realArray.reduce((res, child) => {
-          return reduceShadowDOM(res, child, parent);
-        }, []);
-      }
-    } else {
-      if (node.nodeType === 1) {
-        retVal = createNode(node, parent, shadowId);
-        realArray = Array.from(node.childNodes);
-        retVal.children = realArray.reduce((res, child) => {
-          return reduceShadowDOM(res, child, retVal);
-        }, []);
-
-        return [retVal];
-      } else if (node.nodeType === 3) {
-        // text
-        return [createNode(node, parent)];
-      }
-      return undefined;
+  }
+
+  if (
+    nodeName === 'content' &&
+    typeof node.getDistributedNodes === 'function'
+  ) {
+    realArray = Array.from(node.getDistributedNodes());
+    return createChildren(realArray, parent, shadowId);
+  }
+
+  if (nodeName === 'slot' && typeof node.assignedNodes === 'function') {
+    realArray = Array.from(node.assignedNodes());
+    if (!realArray.length) {
+      // fallback content
+      realArray = getSlotChildren(node);
+    }
+
+    const styl = window.getComputedStyle(node);
+
+    // check the display property. intentionally does not run, see notable information above
```

Comment:
What notable information?

Reply:
At the top of the file is a large comment about why we turned this off (I had to go back in the code timeline to figure out why the comment `intentionally commented out` meant